# Generated for NaoQi method ALAudioPlayer.getVolume
# Returns the volume of the player

---
# Volume of the player - range 0.0 to 1.0.
float32 value


